README.md

```diff
@@ -83,6 +83,7 @@ Install dependencies, build the source files and crank up a server via:
 ```
 git clone git@github.com:mapbox/mapbox-gl-draw.git
 yarn install
+node server.js
```

Comment:
Maybe instead we can add `"start-server": "node server.js"` to `scripts` and then add `start-server` to the `start` script so that `yarn start` will do it all.